configuration/pih/scripts/visit/encounterTypeConfig.js

```diff
@@ -834,6 +834,17 @@ angular.module("encounterTypeConfig", [])
             showOnVisitList: true
         };
 
+        // inpatient vitals
+        encounterTypes["6b1c3929-f23c-423f-8cad-64b17cf10b99"] = {
+            defaultState: "short",
+            shortTemplate: "templates/encounters/vitalsShort.page",
+            longTemplate: "templates/encounters/viewEncounterWithHtmlFormLong.page",
+            templateModelUrl: "/htmlformentryui/htmlform/viewEncounterWithHtmlForm/getAsHtml.action?encounterId={{encounter.uuid}}",
+            icon: "fas fa-fw fa-heartbeat",
+            editUrl: hfeSimpleEditUrl,
+            showOnVisitList: true
+        };
```

Comment:
I think this is identical to the setup for the regular vitals form, which suggests we can get away with using the same encounter type? (I see we already have a separate "vitals with glucose" encounter type that appears the same, but looks like that ship has sailed.